src/aap_eda/utils/__init__.py

```diff
@@ -12,6 +12,16 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import pkg_resources
```

Comment:
@Dostonbek1 This seems to have been deprecated in favor of importlib https://docs.python.org/3/library/importlib.metadata.html#module-importlib.metadata
```
from importlib.metadata import version 
version('psycopg')
'3.2.1'
version('aap-eda')
'0.2.0'
```

Reply:
If the python package is not installed it throws **PackageNotFoundError(name)** Maybe at that point we could directly try loading the pyproject.toml and directly read the version